src/Public/partials/ie-conditionals.php

```diff
@@ -1,5 +1,5 @@
 <!--[if lt IE 9]>
```

Comment:
Only Lesser then IE 9? 
What is with the style Changes for IE 9 They get removed but not added here

Reply:
My Bad dam github UI doesn't show complete File